relationalai-sdk-snowflake/pyproject.toml

```diff
@@ -0,0 +1,49 @@
+[build-system]
+requires = ["setuptools", "setuptools-scm"]
+build-backend = "setuptools.build_meta"
+
+[tool.setuptools.packages.find]
+where = ["."]
+include = ["relationalai"]
+namespaces = true
+
+[project]
+name = "relationalai-sdk-snowflake"
```

Comment:
imo this would read better as relationalai-snowflake-sdk

Reply:
That was how I called it initially. But then realized that our regular Python SDK was called rai-sdk (and should probably be renamed to relationalai-sdk in the future) and thought the snowflake version should just have the postfix. Also, imho, putting relationalai and snowflake together makes it kind of harder to understand that it's relationalai's sdk and not snowflake's.

@wienleung @torkins what's your opinion on that? (although I've already asked Patrick on the Slack and he prefers relationalai-sdk-snowflake

Reply:
don't have a strong opinion, but this seems consistent with our other SDK naming. If I think of seeing this as a package in a directory, I think the purpose is clear enough